File: Zipping and Unzipping files in .NET application/[C#]-Zipping and Unzipping files in .NET application/C#/CreateNewArchive.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace ArchiveManager
{
    public partial class CreateNewArchive : Form
    {
        public CreateNewArchive()
        {
            InitializeComponent();

            cbxFileFormat.SelectedIndex = 0;
        }

        public ArchiverType Type
        {
            get { return (ArchiverType) cbxFileFormat.SelectedIndex; }
        }
    }
}

File: Zipping and Unzipping files in .NET application/[VB]-Zipping and Unzipping files in .NET application/VB/CreateNewArchive.vb
Imports System.ComponentModel
Imports System.Text

Namespace ArchiveManager
	Partial Public Class CreateNewArchive
		Inherits Form
		Public Sub New()
			InitializeComponent()

			cbxFileFormat.SelectedIndex = 0
		End Sub

		Public ReadOnly Property Type() As ArchiverType
			Get
				Return CType(cbxFileFormat.SelectedIndex, ArchiverType)
			End Get
		End Property
	End Class
End Namespace